fbw-a32nx/src/systems/instruments/src/EFB/Ground/Pages/FuelPage.tsx

```diff
@@ -442,17 +446,27 @@ export const FuelPage = () => {
                     </div>
 
                     <div
-                        className={`flex justify-center items-center w-20 ${formatRefuelStatusClass()} bg-current`}
+                        className={`flex w-20 items-center justify-center ${formatRefuelStatusClass()} bg-current`}
                         onClick={() => switchRefuelState()}
                     >
-                        <div className={`${airplaneCanRefuel() ? 'text-white' : 'text-theme-unselected'}`}>
-                            <PlayFill size={50} className={refuelStartedByUser ? 'hidden' : ''} />
-                            <StopCircleFill size={50} className={refuelStartedByUser ? '' : 'hidden'} />
-                        </div>
+                        {!gsxFuelSyncEnabled && (
+                            <div className={`${airplaneCanRefuel() ? 'text-white' : 'text-theme-unselected'}`}>
+                                <PlayFill size={50} className={refuelStartedByUser ? 'hidden' : ''} />
+                                <StopCircleFill size={50} className={refuelStartedByUser ? '' : 'hidden'} />
+                            </div>
+                        )}
+                        {gsxFuelSyncEnabled && (
+                            <div className={`${airplaneCanRefuel() ? 'text-white' : 'text-theme-unselected'}`}>
+                                <PlayFill size={50} className={refuelStartedByUser || refuelRate !== '2' ? 'hidden' : ''} />
+                                <StopCircleFill size={50} className={refuelStartedByUser && refuelRate === '2' ? '' : 'hidden'} />
+                                <LockFill size={48} className={!gsxRefuelActive() && refuelRate !== '2' ? '' : 'hidden'} />
+                                <IconUpload size={50} className={gsxRefuelActive() ? '' : 'hidden'} />
```

Comment:
**suggestion:** I find those icons not super clear. We should come up with better UI for this

Reply:
Hmm - maybe the [Arrow repeat](https://icons.getbootstrap.com/icons/arrow-repeat/) or [Bootstrap Reboot](https://icons.getbootstrap.com/icons/bootstrap-reboot/) Icon instead of IconUpload?

Reply:
If it is more about the Lock Icon (it is a bit disproportional, is it?), maybe using the same Icon in the normal and -fill Variant? For Example:
GSX Refuel not called = [Gear](https://icons.getbootstrap.com/icons/gear/)
GSX Refuel called => [Gear fill](https://icons.getbootstrap.com/icons/gear-fill/)

I mean we could also just hide the complete Button (like it is done on the Payload Page)? My main Motivation with that is to not show Play & Stop when there is nothing to Start/Stop for the User there. Since the Interface for that would be the GSX Menu (or with lucky38i's PR 8225 the Services Page maybe in the Future).
Sometimes less is more 😅

Reply:
How's that?

![image](https://github.com/flybywiresim/aircraft/assets/78614902/a050539d-2ac5-4c9d-8fc0-bcb6d106072c)

=>

![image](https://github.com/flybywiresim/aircraft/assets/78614902/22b235a8-6f43-448e-8cce-b82c6a70f760)

=> 

![image](https://github.com/flybywiresim/aircraft/assets/78614902/30b292cd-8270-490d-a5b5-1ad7438dbefd)

Reply:
Can we not show the same icons, but with the button grayed out?

Something like `opacity-50`

Reply:
You mean just like before Play/Stop just with another opacity when GSX Sync is selected/active?
Sure, it was just an additional Proposal to make it clearer to the User when an Interaction is possible.

Main Motivation is the Automatic Start and Instant Refuel allowed for the GSX Fuel Sync. So together with the Automatic Weights Import I've also prepared most GSX Users would not pay much Attention to that Page anymore anyways 😅

Reply:
```
                    <div
                        className={`flex w-20 items-center justify-center ${formatRefuelStatusClass()} bg-current${(gsxFuelSyncEnabled && refuelRate !== '2' ? ' opacity-50' : '')}`}
                        onClick={() => switchRefuelState()}
                    >
                        <div className={`${airplaneCanRefuel() ? 'text-white' : 'text-theme-unselected'}`}>
                            <PlayFill size={50} className={refuelStartedByUser ? 'hidden' : ''} />
                            <StopCircleFill size={50} className={refuelStartedByUser ? '' : 'hidden'} />
                        </div>
                    </div>
```

![image](https://github.com/flybywiresim/aircraft/assets/78614902/d4dea8dd-5443-4cbc-80b9-540eb3c7f44d)

=>

![image](https://github.com/flybywiresim/aircraft/assets/78614902/55d784f9-d564-47da-a1fa-628693aa95e4)

=> 

![image](https://github.com/flybywiresim/aircraft/assets/78614902/c2138a99-fac2-415a-8e34-4ae4a4f21462)

Reply:
Changes pushed according to the Screenshots shown in Discord:
https://discord.com/channels/738864299392630914/1100470848533577761/1196212496583897252